libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -267,7 +268,7 @@ userdata mavlink_mission_item_int_t field command uint16_t read write 0 UINT16_M
 -- userdata mavlink_mission_item_int_t field target_component uint8_t read write 0 UINT8_MAX
 userdata mavlink_mission_item_int_t field frame uint8_t read write  0 UINT8_MAX
 userdata mavlink_mission_item_int_t field current uint8_t read write  0 UINT8_MAX
--- userdata mavlink_mission_item_int_t field autocontinue uint8_t read write  0 UINT8_MAX
```

Comment:
We don't do anything with this, it's always 1 internally and we only do anything with it for the sake of the GCS/packet, and even then we don't serialize that data. I'd drop this from the binding, as it just adds to memory consumption without giving any useful features.